index.html

```diff
@@ -0,0 +1,814 @@
+          >
+            <g clip-path="url(#clip0_964_6233)">
+              <path
+                d="M5.33321 14.9336C5.92231 14.9336 6.39987 14.4561 6.39987 13.867C6.39987 13.2779 5.92231 12.8003 5.33321 12.8003C4.7441 12.8003 4.26654 13.2779 4.26654 13.867C4.26654 14.4561 4.7441 14.9336 5.33321 14.9336Z"
+                fill="#737373"
+              />
+              <path
+                d="M12.8001 14.9336C13.3892 14.9336 13.8668 14.4561 13.8668 13.867C13.8668 13.2779 13.3892 12.8003 12.8001 12.8003C12.211 12.8003 11.7335 13.2779 11.7335 13.867C11.7335 14.4561 12.211 14.9336 12.8001 14.9336Z"
+                fill="#737373"
+              />
+              <path
+                d="M14.9333 2.66678H3.104L2.66667 0.426775C2.64173 0.304495 2.57471 0.19483 2.47726 0.11687C2.37981 0.0389102 2.25811 -0.00240773 2.13333 0.000108501H0V1.06678H1.696L3.73333 11.3068C3.75827 11.4291 3.82529 11.5387 3.92274 11.6167C4.02019 11.6946 4.1419 11.736 4.26667 11.7334H13.8667V10.6668H4.704L4.26667 8.53344H13.8667C13.99 8.53646 14.1105 8.49665 14.2077 8.4208C14.305 8.34496 14.3729 8.23775 14.4 8.11744L15.4667 3.31744C15.4845 3.23831 15.4841 3.15614 15.4655 3.07719C15.4468 2.99824 15.4103 2.9246 15.3589 2.86185C15.3075 2.79911 15.2424 2.74893 15.1687 2.71512C15.0949 2.68132 15.0144 2.66478 14.9333 2.66678ZM13.44 7.46678H4.064L3.31733 3.73344H14.2667L13.44 7.46678Z"
+                fill="#737373"
+              />
+            </g>
+            <defs>
+              <clipPath id="clip0_964_6233">
+                <rect width="16" height="14.9334" fill="white" />
+              </clipPath>
+            </defs>
+          </svg>
+        </a>
+        <a href="" class="navbar__icon"
+          ><svg
+            width="24"
+            height="14"
+            viewBox="0 0 24 14"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M0.571472 0H23.4286V2.28571H0.571472V0ZM6.28576 5.71429H23.4286V8H6.28576V5.71429ZM13.4286 11.4286H23.4286V13.7143H13.4286V11.4286Z"
+              fill="#737373"
+            />
+          </svg>
+        </a>
+      </div>
+    </nav>
+    <div class="hero">
+      <h1 class="hero__title">Experts are here solve your business problem.</h1>
+      <p class="hero__desc">
+        We know how large objects will act, but things on a small scale just do
+        not act that way.
+      </p>
+      <div class="hero__links">
+        <a class="btn btn--secondary" href="">Get Quote Now</a>
+        <a class="btn btn--outlined" href="">Learn More</a>
+      </div>
+    </div>
+    <div class="why">
+      <div class="why__header">
+        <h4 class="why__title">Why choose us</h4>
+        <p class="why__subtitle">
+          Problems trying to resolve the conflict between the two major realms
+          of Classical physics: Newtonian mechanics
+        </p>
+      </div>
+      <div class="why__content">
+        <div class="whycard__item">
```

Comment:
where do these cards with whycard__item belong to. they should have parent container of "whycard"